frontend/src/screens/Intro.tsx

```diff
@@ -81,7 +81,7 @@ export function Intro() {
               </div>
               <div className="mt-20">
                 <Button onClick={() => api?.scrollNext()} size="lg">
-                  Get Started
+                  Next
```

Comment:
@reneaaron I changed this so we don't have two "Get Started" - raised by @bumi a while back